spec/Procedures.tex

```diff
@@ -1224,24 +1225,47 @@ \subsection{Determining More Specific Functions}
 mapping to $F_2$ and none of the legal argument mappings to $F_2$ is a
 more specific argument mapping than the corresponding legal argument
 mapping to $F_1$, then $F_1$ is more specific.
-\item
-If at least one of the legal argument mappings to $F_2$ is a {\em more
-specific argument mapping} than the corresponding legal argument
-mapping to $F_1$ and none of the legal argument mappings to $F_1$ is a
-more specific argument mapping than the corresponding legal argument
-mapping to $F_2$, then $F_2$ is more specific.
+
+\item Similarly, if at least one argument mapping to $F_2$ is {\em more
```

Comment:
I think this should go back to matching the pattern of the rest of the list items - worded the same as the previous point but with F_1 and F_2 swapped.